### PR TITLE
add missing return in context

### PR DESCRIPTION
**Pull request by mvpatel2000**
as title

### CODE REVIEW

**Comment by mvpatel2000** [comment]
Automerge enabled. Ty @dakinggg  for catching this!!

**Review by dakinggg — APPROVED** [review]
LGTM

**Comment by dakinggg** [comment]
@mvpatel2000 I guess I can't request a review from the PR opener, but consider this a review request to see if my tests look good :)

File: tests/utils/test_dist.py [hunk 1]
```diff
@@ -1,16 +1,29 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
 
+from unittest.mock import patch
+
```
Comment (mvpatel2000) [inline tests/utils/test_dist.py:5]: very minor nit: can we get rid of this new line and put in same import block?

**Review by mvpatel2000 — COMMENTED** [review]
This is amazing! TYSM :)

File: tests/utils/test_dist.py (hunk 1 above)
Comment (dakinggg) [reply tests/utils/test_dist.py:5]: Actually no. The auto formatter is doing this.

Comment (mvpatel2000) [reply tests/utils/test_dist.py:5]: 😭 ok will merge then

**Pull request merged**